### PR TITLE
Start improving gene annotation track creation docs

### PR DESCRIPTION
**Pull request by nvictus**
Steps still need better descriptions.

Fixes:

* `gzcat` and `zgrep` are esoteric (installing GNU utils on Mac). `zcat` should be available on most linux/mac environments.

* what were those virtualenv-wrapper and s3 commands doing there?

* provided download link for `exonU.py`

### CODE REVIEW

File: docs/data_preparation.rst [hunk 1]
```diff
@@ -455,37 +420,29 @@ Processing
...
+        --assembly $ASSEMBLY \
+        --output-file $DATADIR/$ASSEMBLY/gene-annotations-${ASSEMBLY}.db \
         --delimiter $'\t' \
-        ~/data/genbank-data/${ASSEMBLY}/geneAnnotationsExonUnions.bed 
+        $DATADIR/$ASSEMBLY/geneAnnotationsExonUnions.bed 
 
-    aws s3 cp ~/data/tiled-data/gene-annotations-${ASSEMBLY}.db \
-        s3://pkerp/public/hg-server/data/${ASSEMBLY}/
 
 Importing into HiGlass
```
Comment (nvictus) [inline docs/data_preparation.rst]: Why are these authenticated form requests to `higlass.io` here?

Comment (pkerpedjiev) [reply docs/data_preparation.rst]: They shouldn't be. Thanks for deleting them 😬

File: docs/data_preparation.rst [hunk 2]
```diff
@@ -455,37 +420,29 @@ Processing
...
-    ASSEMBLY=mm9
+.. code-block:: bash
 
     clodius aggregate bedfile \
-        --max-per-tile 20 --importance-column 5 \
-        --assembly ${ASSEMBLY} \
-        --output-file ~/data/tiled-data/gene-annotations-${ASSEMBLY}.db \
+        --max-per-tile 20 \
+        --importance-column 5 \
+        --assembly $ASSEMBLY \
```
Comment (pkerpedjiev) [inline docs/data_preparation.rst]:
```suggestion
        --chromsizes-filename assembly.chromSizes \
```

File: docs/data_preparation.rst (hunk 1 above)
Comment (nvictus) [reply docs/data_preparation.rst]: I haven't yet :P. I noticed them after making the PR...

Comment (nvictus) [reply docs/data_preparation.rst]: Now they're gone :)

**Pull request merged**